src/tr2/game/game_flow/sequencer_events.c

```diff
@@ -200,7 +200,7 @@ static DECLARE_GF_EVENT_HANDLER(M_HandleDisableFloor)
 {
     GF_COMMAND gf_cmd = { .action = GF_NOOP };
     if (seq_ctx != GFSC_STORY) {
-        g_GF_NoFloor = (int16_t)(intptr_t)event->data;
+        Room_SetNoFloorHeight((int16_t)(intptr_t)event->data);
```

Comment:
Maybe `SetAbyssHeight`? or `SetVoidHeight`. I'm also perfectly fine with the current naming.

Reply:
Thank, I think `Abyss` is a good choice; `NoFloor` was a bit awkward.